src/clj/rems/css/styles.clj

```diff
@@ -685,6 +685,7 @@
     ;; make fieldset legends look the same as normal labels
     [:legend {:font-size "inherit"}]]
    [:.application-field-label {:font-weight "bold"}]
+   [:.administration-field-label {:font-weight "bold"}]
```

Comment:
```font-weight: 600``` would be better imho

Reply:
Looks like we only use normal (400), bold (700) and 300 so far, and only import web fonts for these weights. Why 600 here?